target/linux/ramips/base-files/etc/hotplug.d/firmware/10-rt2x00-eeprom

```diff
@@ -41,6 +54,11 @@ board=$(board_name)
 case "$FIRMWARE" in
 "soc_wmac.eeprom")
 	case $board in
+	dlink,dwr-116-a)
+		wifi_mac=$(jboot_config_read -m -i /dev/mtd5 -o 0xE000)
```

Comment:
For the `wifi_mac` I would implement another `jboot_xxxxxx_extract` function that extracts the *WAN* MAC from the "config" partition, and adds 1 to it (with the help of `. /lib/functions/system.sh`).

I would also save the `0xE000` offset to a local variable to avoid writing it twice. :)

Reply:
> ... I would implement another `jboot_xxxxxx_extract` function ...

In the previous version of `jboot-tools` there was a functionality like that? Wasn't it?